src/binder/bindertracing.cpp

```diff
@@ -68,6 +77,71 @@ namespace
             &activityId);
 #endif // FEATURE_EVENT_TRACE
     }
+
+    void PopulateBindRequest(/*inout*/ BinderTracing::AssemblyBindOperation::BindRequest &request)
```

Comment:
I'm going to need the ALC name for the resolving event too, so maybe this can be factored out in a utility function that takes an AssemblySpec and writes/returns a SString?

Reply:
(Filling out a SString might be better because there's a version that initially allocate it inline with the object, avoiding heap allocations.)

Reply:
Do you actually need the ALC name on the unmanaged side? If you deal with the ALC on managed side only, it will save you a few allocations.

Reply:
I currently need this from the non-managed side, unfortunately.  However, since in most cases, the ALC name from a resolving event will be used for the load/bind event, that string can be reused somehow.  Optimization/cleanup for a later pass, maybe.

Reply:
What do you need it for from the non-managed side for?